README.md

```diff
@@ -277,4 +324,26 @@ A successful request will return the following JSON encoded response
 }
 ```
 
-
+## Error codes
+
+| Code | Description                    |
+|------|--------------------------------|
+| 1000 | Too low airtime amount         |
+| 1001 | Too big airtime amount         |
+| 1002 | Unsupported country            |
+| 1003 | Unsupported carrier            |
+| 1004 | Service unavailable            |
+| 1005 | Invalid airtime amount format  |
+| 1006 | Invalid recipient phone number |
+| 1007 | Invalid strategy               |
+| 1008 | Invalid top-up request ID      |
+| 1009 | Insufficient funds             |
+
+#### All validation or processing error responses will have the following format and HTTP status code 400 (Bad request)
```

Comment:
Will it be 400 when service is unavailable as well, I would think it should be 500?